db/migrate/20190718201250_create_landmarks.rb

```diff
@@ -1,16 +0,0 @@
-class CreateLandmarks < ActiveRecord::Migration[5.2]
```

Comment:
In the future, it's better to write another migration that drops the table instead of deleting a migration. Deleting a migration means that everyone (including production) will have to completely drop the database to get the right schema.